pandas/core/series.py

```diff
@@ -1429,8 +1429,28 @@ def mode(self):
         # TODO: Add option for bins like value_counts()
         return algorithms.mode(self)
 
-    @Appender(base._shared_docs['unique'] % _shared_doc_kwargs)
     def unique(self):
+        """
+        Return unique values in the object. Uniques are returned in order
+        of appearance, this does NOT sort. Hash table-based unique.
```

Comment:
I'm not sure hash table-based unique will be meaningful to many users.

Reply:
agree but that's the same in pandas.unique. I think it makes sense to people who understand data structure, and for people who don't understand it there is enough clarification around it. It's important to let people to know it's hash table so no sorting.